### PR TITLE
Fix/remove tickets rsvp blocks

### PR DESCRIPTION
**Pull request by paulmskim**
For tickets:
- https://central.tri.be/issues/ 119662
- https://central.tri.be/issues/ 119663

Depends on and requires the following PRs to be merged:
- https://github.com/moderntribe/event-tickets/pull/1033
- https://github.com/moderntribe/event-tickets/pull/1026

### CODE REVIEW

**Comment by tr1b0t** [comment]
Closing this Pull Request because it is pull requested against the wrong branch. The Central ticket is for *G19.01*, but this was pull requested against *release/M19.01*.

File: src/views/blocks/tickets/content-inactive.php [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+ *
+ * See more documentation about our Blocks Editor templating system.
+ *
+ * @link {INSERT_ARTCILE_LINK_HERE}
+ *
+ * @version 4.9
+ *
+ */
+
+$message = $this->get( 'sale_past' ) ? esc_html__( 'Tickets are no longer available', 'event-tickets' ) : esc_html__( 'Tickets are not yet available', 'event-tickets' );
```
Comment (sc0ttkclark) [inline src/views/blocks/tickets/content-inactive.php]: Use __()

File: src/views/blocks/tickets/content-inactive.php [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+?>
+<div
+	class="tribe-block__tickets__item__content tribe-block__tickets__item__content--inactive"
+>
+	<?php echo $message ?>
```
Comment (sc0ttkclark) [inline src/views/blocks/tickets/content-inactive.php]: Use esc_html on $message for late escaping

File: src/views/blocks/tickets.php [hunk 3]
```diff
@@ -20,10 +20,30 @@
...
+$has_tickets_on_sale = ! empty( count( $tickets_on_sale ) );
+
+if ( ! $has_tickets_on_sale ) {
+	$sale_past = ! empty( count( $tickets ) );
+	$timestamp = current_time( 'timestamp' );
+
+	foreach ( $tickets as $ticket ) {
+		$sale_past = ( $sale_past && $ticket->date_is_later( $timestamp ) );
+	}
+}
```
Comment (sc0ttkclark) [inline src/views/blocks/tickets.php]: Can this be moved to a method and pull it out of the view?

File: src/modules/blocks/rsvp/container.js [hunk 4]
```diff
@@ -54,7 +54,7 @@ const setInitialState = ( dispatch, ownProps ) => () => {
 	dispatch( thunks.getRSVP( postId ) );
```
Comment (sc0ttkclark) [inline src/modules/blocks/rsvp/container.js]: If thunks was renamed to actions, should this be changed too?

Comment (sc0ttkclark) [reply src/modules/blocks/rsvp/container.js]: thunks is still used above in the import, so maybe it's just the method that got moved

File: src/views/blocks/tickets/item-inactive.php [hunk 5]
```diff
@@ -0,0 +1,30 @@
...
+$classes   = array(
+	'tribe-block__tickets__item',
+	'tribe-block__tickets__item--inactive',
+);
+
+$context = array(
+	'sale_past' => $sale_past,
+);
+?>
+<div class="<?php echo implode( ' ', get_post_class( $classes ) ); ?>">
```
Comment (sc0ttkclark) [inline src/views/blocks/tickets/item-inactive.php]: Just noticed this, it needs to have esc_attr()

Comment (sc0ttkclark) [reply src/views/blocks/tickets/item-inactive.php]: Can you also add esc_attr() anywhere else you see implode() happening in these block views? I see it in src/views/blocks/tickets/item.php too

**Comment by sc0ttkclark** [comment]
Needs conflict resolution

**Pull request merged**